<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptedFileDialog</class>
 <widget class="QDialog" name="EncryptedFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EncryptedFileDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="SourceFileLabel">
         <property name="text">
          <string>SourceFile:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SourceFileLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="InputFilePushButton">
         <property name="text">
          <string>InputFile</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="EncryptionPushButton">
         <property name="text">
          <string>Encryption</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="DecryptionPushButton">
         <property name="text">
          <string>Decryption</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
